pysat/model_utils.py

```diff
@@ -348,11 +352,11 @@ def collect_inst_model_pairs(start=None, stop=None, tinc=None, inst=None,
                             matched_inst.data = inst.data.isel(im)
                     else:
                         if inst.pandas_format:
-                            matched_inst.data = matched_inst.data.append( \
+                            matched_inst.data = matched_inst.data.append(
```

Comment:
Is it now allowable to change the indentation without specifying that is what is being done?

Reply:
changing back